vagrant/envvars.conf

```diff
@@ -0,0 +1,12 @@
+# This section gets appended to the kive user's .bash_profile
+# and to /etc/sysconfig/httpd
+export KIVE_DB_NAME=kive
+export KIVE_DB_USER=kive
+export KIVE_DB_HOST=head
+export KIVE_DB_PASSWORD=YZcGRH8AZAj6VAF6qj8KOy
```

Comment:
If we're going to include a password in source control, we should probably have a comment explaining why. One strategy I've used in the past is to generate a random password during dev/test setup and write it into a configuration file.